### ISSUE TITLE
Full Workspace Backup

### ISSUE THREAD

**Issue by svenefftinge** [gitpod-io/gitpod#7719]
Full workspace backup has been lying around for some time now, with no real strategy on rolling out. We should allow starting full workspace backups using the volume mount approach so that we can allow users to opt-in to use this feature through a user setting. 

Eventually we should deprecate and remove the old mode.

Rough Plan:

**End of April**: Feature Preview

**End of May**: GA

**Comment by shaal** [comment]
Is there any update on this issue?

**Comment by sagor999** [comment]
This is superseded by this epic: https://github.com/gitpod-io/gitpod/issues/7901
@atduarte should this one be closed now?

**Comment by atduarte** [comment]
I though full workspace backup was out of scope for #7901 [[1](https://www.notion.so/gitpod/Using-PVC-Snapshots-for-workspace-storage-management-fd932e28a549409581c21d6f0a7953c8#869fb3486c004d1192b70692b5fc490b)] and that the PVC would only contain the `/workspace` folder 🤔 

If #7901 solves this, I would leave this open as a issue but reference the durability epic.

**Comment by sagor999** [comment]
@atduarte you are right, it is indeed out of scope for that epic. I got confused a bit here. :sweat_smile:

**Comment by a-b** [comment]
I'm wondering if some workaround is possible. What if we link folders like `/home/linuxbrew/.linuxbrew` to `/workspace`? Perhaps this could have some reflection in `.gitopod.yml` ?
